server/routes/api/teams/teams.test.ts

```diff
@@ -287,7 +287,7 @@ describe("#team.update", () => {
       body: {
         token: admin.getJwtToken(),
         id: collection.id,
-        permission: "",
```

Comment:
I hope this is fine and nobody's sending `permission` as an empty string :crossed_fingers: